src/sentry/tasks/deletion.py

```diff
@@ -267,6 +267,52 @@ def delete_group(object_id, transaction_id=None, **kwargs):
         )
 
 
+@instrumented_task(
+    name='sentry.tasks.deletion.delete_groups',
+    queue='cleanup',
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES
+)
+@retry(exclude=(DeleteAborted, ))
+def delete_groups(object_ids, transaction_id=None, eventstream_state=None, **kwargs):
+    from sentry import deletions, eventstream
+    from sentry.models import Group
+
+    if not object_ids:
+        return
```

Comment:
I think there would be a bug if we end up in this conditional? We'd have (probably) already sent the `start` message and potentially never end up sending the `end` message.